packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -50,7 +50,8 @@ const PARAMETERS_FILENAME = 'parameters.json';
 const SCHEMA_FILENAME = 'schema.graphql';
 const SCHEMA_DIR_NAME = 'schema';
 const PROVIDER_NAME = 'awscloudformation';
-const LAYER_MAPPING_URL = 'https://amplify-rds-layer-resources.s3.amazonaws.com/rds-layer-mapping.json';
+const LAYER_MAPPING_S3_BUCKET = 'amplify-rds-layer-resources';
+const USE_BETA_SQL_LAYER = 'use-beta-sql-layer';
```

Comment:
Here and throughout: Put these in ResourceConstants alongside the other patching lambda constants. That way we centralize the value and can access it easily from tests that don't need to assert the actual constant values

Reply:
I've reused the existing constant for the SQL Layer S3 manifest bucket. I left the CLI Gen1 option in the same file as this is not related to the transformers.